src/engine/bank01.asm

```diff
@@ -7172,7 +7172,45 @@ Func_6cfa: ; 6cfa (1:6cfa)
 	ret
 ; 0x6d3f
 
-	INCROM $6d3f, $6d84
+Func_6d3f: ; 6d3f (1:6d3f)
+	or a
+	bit 7, [hl]
+	ret z
+	push hl
+	bit 6, [hl]
+	ld a, $0a
+	ld hl, $26      ; copies from the byte of the damage due to poison by 10
```

Comment:
Instead of hand-writing a comment, you can remove the hard-coded value of `$26` since it is the text id for `Received10DamageDueToPoisonText` by replacing this whole line with:
```
ldtx hl, Received10DamageDueToPoisonText
```
(and likewise for text `$27` below)